staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/plugins.go

```diff
@@ -26,8 +26,13 @@ import (
 	"github.com/golang/glog"
 
 	"k8s.io/apiserver/pkg/storage/value/encrypt/envelope"
+	"k8s.io/apiserver/plugin/pkg/kmsprovider/vault"
```

Comment:
This dependency shouldn't be here.  The point of plugins is to allow the apiserver to decide what to register.